systemd_ctypes/event.py

```diff
@@ -84,8 +84,23 @@ def new_event_loop(self):
 def run_async(main):
     asyncio.set_event_loop_policy(EventLoopPolicy())
 
-    if sys.version_info >= (3, 7, 0):
-        asyncio.run(main)
-    else:
+    polyfill = sys.version_info < (3, 7, 0) and not hasattr(asyncio, 'run')
```

Comment:
OK, but in the face of the possibility of backports, I'd rather just test the presence of each polyfilled method individually (instead of the version check), and drop the asserts.

Reply:
I considered that, but it made the code a fair bit more complicated (particularly with respect to backing the changes out again later).  That's an argument towards making the polyfills permanent, though.